ACKategoriesCoreTests/Random/StringRandomTests.swift

```diff
@@ -0,0 +1,26 @@
+import XCTest
+import Foundation
+
+final class StringRandomTests: XCTestCase {
+    func testRandomStringContainsOnlyAllowedLetters() {
+        DispatchQueue.concurrentPerform(iterations: 50) { _ in
+            let randomString = String.random(allowedLetters: "ad")
+            XCTAssertTrue(randomString.allSatisfy("ad".contains))
+        }
+    }
+    
+    func testLengthOfStringIsSmallerThanGivenValue() {
+        DispatchQueue.concurrentPerform(iterations: 50) { _ in
+            let randomString = String.random(minLength: 0, maxLength: 2)
+            print(randomString.count)
```

Comment:
Probably for testing purposes. Can be deleted?

Reply:
Yep 🙈